commands/create/cluster/command.go

```diff
@@ -102,6 +106,16 @@ type creationResult struct {
 	HasErrors bool
 }
 
+// JSONOutput contains the fields included in JSON output of the create cluster command when called with json output flag
+type JSONOutput struct {
+	// cluster ID
```

Comment:
We don't really enforce Go Doc comments where the identifier is the first word of the comment, but would be nice to stick to it for consistency here: like maybe `ID of the cluster`. Same with `Result` below.